src/components/pages/budget/BudgetList.vue

```diff
@@ -43,6 +43,12 @@
                   />
                 </div>
               </th>
+              <th class="datatable-row-header duration-header month">
+                {{ $t('budget.fields.base_salary') }}
+              </th>
+              <th class="datatable-row-header base-salary-header month">
+                {{ $t('budget.fields.duration') }}
+              </th>
```

Comment:
The class names are inverted: duration-header & base-salary-header

```suggestion
              <th class="datatable-row-header base-salary-header month">
                {{ $t('budget.fields.base_salary') }}
              </th>
              <th class="datatable-row-header duration-header month">
                {{ $t('budget.fields.duration') }}
              </th>
```